apps/api/src/types/person_role/dto/order-person_role.input.ts

```diff
@@ -4,7 +4,8 @@ import {NumberComparisonInput} from "../../../gql/number-comparison.input";
 
 enum PersonRoleOrderableFields {
     id = "id",
-    startTime = "startTime"
+    startTime = "startTime",
+    priority = "priority",
```

Comment:
So it shouldn't be sorted by priority or it currently can't be?

Reply:
`PersonRole` has no `priority` field, so it can't be sorted by it.

https://github.com/rpitv/glimpse/blob/master/packages/prisma/schema.prisma#L232-L245